galata/test/jupyterlab/cells.test.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, galata, test } from '@jupyterlab/galata';
+import * as path from 'path';
+
+const fileName = 'simple_notebook.ipynb';
+
+test.describe('Cells', () => {
+  test.beforeEach(async ({ page, request, tmpPath }) => {
+    const contents = galata.newContentsHelper(request);
+    await contents.uploadFile(
+      path.resolve(__dirname, `./notebooks/${fileName}`),
+      `${tmpPath}/${fileName}`
+    );
+    await contents.uploadFile(
+      path.resolve(__dirname, './notebooks/WidgetArch.png'),
+      `${tmpPath}/WidgetArch.png`
+    );
```

Comment:
Possibly not strictly necessary for this test, but also fine to keep in case if this notebook were to be used for other tests.